pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/BacklogQuota.java

```diff
@@ -42,7 +42,7 @@ public interface BacklogQuota {
      *
      * @return quota limit in bytes
      */
-    long getLimitSize();
+    long getLimitSizeInBytes();
```

Comment:
This will break existing admin java clients. 
Do we have a strong reason to change it ? If so, we should mark this as deprecated and add a new method

Reply:
Also for completeness we should change the builder method https://github.com/apache/pulsar/pull/19118/files#diff-6818b3dc13db4b7fd2504c6638af715ccf140d0c6d4c17a80420c80052889fd3L57